libs/common/src/main/kotlin/vedtak/VedtakDto.kt

```diff
@@ -39,24 +33,20 @@ data class Periode(
     }
 }
 
-data class BilagMedSammendrag<T>(
-    val bilag: ObjectNode,
-    val sammendrag: T
-)
-
 data class VedtakFattet(
     val ansvarligSaksbehandler: String,
     val ansvarligEnhet: String,
-    val tidspunkt: ZonedDateTime
+    val tidspunkt: Tidspunkt
 )
 
-data class Beregningsperiode(
-    val periode: Periode,
-    val beloep: BigDecimal
+data class Attestasjon(
+    val attestant: String,
+    val attesterendeEnhet: String,
+    val tidspunkt: Tidspunkt
 )
 
 data class Utbetalingsperiode(
-    val id: Long,
+    val id: Long = 0L,
```

Comment:
Skeptisk til å gi denne en standardverdi. Er det kun for testing? Hvis så ville jeg fjernet det.

Reply:
den var tidligere alltid satt til 0 før man lagret den ned - hvor den så fikk en verdi.. Endrer den til å være nullable, slik at det feltet kun får en verdi etter at utbetalingsperioden er lagret ned. Ikke super-fan av den fremgangsmåten, men gir mer mening enn at den settes til 0 vil jeg tro